config/alpine/waagent.conf

```diff
@@ -75,7 +75,11 @@ OS.OpensslPath=None
 OS.SshDir=/etc/ssh
 
 # Enable or disable goal state processing auto-update, default is enabled
-# AutoUpdate.Enabled=y
+# When turned off, it remains on latest version installed on the vm
+# Added this new option AutoUpdate.UpdateToLatestVersion in place of AutoUpdate.Enabled, and encourage users to transition to this new option
+# See wiki[https://github.com/Azure/WALinuxAgent/wiki/FAQ#autoupdateenabled-vs-autoupdateupdatetolatestversion] for more details
+# TODO: Update the wiki link and point to readme page or public facing doc
```

Comment:
Ideally, I want to point to readme link for flag definition but those changes not there in master yet